packages/truffle-environment/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "truffle-environment",
+  "version": "1.0.0",
+  "description": "This package contains the code needed to determine environment-specific data",
+  "main": "index.js",
+  "author": "Faina Shalts",
```

Comment:
I think the dependencies for the pkg need to be added here (i.e. `ganache-core`, `node-ipc`, `web3`, etc)

Reply:
Oh, also believe we want the initial pkg version to be `"0.1.0"`